lib/manageiq/environment.rb

```diff
@@ -121,6 +121,46 @@ def self.bundler_version
       File.read(gemfile).match(/gem\s+['"]bundler['"],\s+['"](.+?)['"]/)[1]
     end
 
+    def self.warn_if_branch_out_of_date
+      return if %w(production test).include?(ENV['RAILS_ENV'].to_s)
+
+      Dir.chdir(APP_ROOT) do
+        commits =
+          if upstream_tracking_branch.empty?
+            system!("git fetch upstream")
```

Comment:
Instead of hardcoding upstream, should we look at the remotes and find which one is tied to `ManageIQ/manageiq`?

Reply:
yeah, @agrare was thinking that too.  If it helps, i can do that.